test/services/supplementary-billing/fetch-licences.service.test.js

```diff
@@ -28,29 +27,44 @@ describe('Fetch Licences service', () => {
   })
 
   describe('when there are licences for the matching region', () => {
-    describe('that are flagged to be included in supplementary billing', () => {
+    describe('that are flagged to be included in supplementary billing.', () => {
```

Comment:
Ideally, the context block descriptions should 'flow', following on from the previous to form documentation on how the app is expected to behave.

This provides the context for subsequent comments, but why we also wouldn't expect to see the description end with '.'

```suggestion
    describe('that are flagged to be included in supplementary billing', () => {
```